apko-publish/action.yaml

```diff
@@ -138,17 +144,18 @@ outputs:
       The digest of the published container image.
 
 runs:
-  using: docker
-  image: "docker://ghcr.io/wolfi-dev/apko:latest"
-  env:
-    # Set up go-containerregistry's GitHub keychain.
-    GITHUB_ACTOR: ${{ inputs.repository_owner }}
-    GITHUB_TOKEN: ${{ inputs.token }}
-    REPOSITORY: ${{ inputs.repository }}
-  entrypoint: /bin/sh
-  args:
-    - '-c'
-    - |
+  using: composite
+  steps:
+  - shell: bash
+    run: |
+      docker run -i --rm --entrypoint /bin/sh \
+        -v $PWD:/workspace \
+        --workdir /workspace \
+        -e "GITHUB_ACTOR=${{ inputs.repository_owner }}" \
+        -e "GITHUB_TOKEN=${{ inputs.token }}" \
+        -e "REPOSITORY=${{ inputs.repository }}" \
```

Comment:
I'm not sure that anything really tested this action in the downstream workflow, but I tried to mirror the other action, with the main exception of these environment variables 🤞 

If folks have ideas for testing this before this merges, I'm open to suggestions, but maybe we should also consider adding some tests to this repo since iterating downstream was pretty painful 😅